### PR TITLE
Explain static method magic in stores

### PR DESCRIPTION
**Pull request by jdlehman**
- Solves Issue #69

### CODE REVIEW

**Comment by troutowicz** [comment]
:thumbsup:

File: docs/stores.md [hunk 1]
```diff
@@ -7,7 +7,9 @@ permalink: /docs/stores/
 
 # Alt Stores
 
-These are the stores returned by [`alt.createStore`](createStore.md), they will not have the methods defined in your StoreModel because flux stores do not have direct setters. However, any static methods defined in your StoreModel will be transferred to this object.
+These are the stores returned by [`alt.createStore`](createStore.md), they will not have the methods defined in your StoreModel because flux stores do not have direct setters. However, any `static` methods defined in your StoreModel will be transferred to this object.
```
Comment (goatslacker) [inline docs/stores.md]: :+1:

File: docs/stores.md [hunk 2]
```diff
@@ -7,7 +7,9 @@ permalink: /docs/stores/
...
+
+**Please Note:** The behavior of `static` methods in alt stores is NOT how `static` methods behave in ES6. Normally `static` methods can be called directly off of the class without creating a new instance of said class, `MyClass.staticMethod()`. In alt, there is magic under the hood that takes the `static` methods you define on a store, and makes them accessible from the store object created by `alt.createStore`. This provides an easy way to add methods that you need to be accessible in your views. If you do not like the magic, or are a fan of being more explicit, you can use [`StoreModel#exportPublicMethods`](createStore.md#storemodelexportpublicmethods) to accomplish the same thing.
```
Comment (goatslacker) [inline docs/stores.md]: Static isn't exclusive to ES6.

Can we reword this to something like:

```txt
Static methods defined on a store model is nothing more than syntactic sugar for exporting the method as a public method of your alt instance. This means that `this` will be bound to the store instance. It is recommended to explicitly export the methods in the constructor using [`StoreModel#exportPublicMethods`](createStore.md#storemodelexportpublicmethods).
```

I feel it's more concise and makes it ES version agnostic.

**Comment by goatslacker** [comment]
As for issue #69 I want to explain all the magic. Not just the static one.

File: docs/stores.md (hunk 2 above)
Comment (troutowicz) [inline docs/stores.md]: > Static methods defined on a store model **are** nothing...

:)

Comment (goatslacker) [inline docs/stores.md]: yup s/is/are

also, great job ending the quote there :P

Comment (troutowicz) [inline docs/stores.md]: My bias towards `exportPublicMethods` got the best of me...

**Comment by jdlehman** [comment]
Also :+1: to explaining ALL the magic.
